share/nisar/defaults/focus.yaml

```diff
@@ -415,11 +415,18 @@ runconfig:
             # cal pulses.
             zero_fill_gaps: True
 
-            # If true, subtract the azimuth mean from the raw data, which is
-            # useful when the caltone is nearly constant in azimuth but not
-            # stationary in range.  Calculating the mean and subtracting it is
-            # done in blocks using rangecomp.block_size
-            nullify_azimuth_mean: True
+            caltone:
+                # If "azimuth_mean", subtract the azimuth mean from the raw
+                # data, which is useful when the caltone is nearly constant in
+                # azimuth but not stationary in range.  Calculating the mean
+                # and subtracting it is done in blocks using
+                # rangecomp.block_size
+                # If "wavelet" then subtract an estimate that varies in 2D.
+                # No correction when null.
+                # wavelet | azimuth_mean | null
+                algorithm: azimuth_mean
```

Comment:
Consider adding another option "auto" here, that would allow the SAS to decide which algorithm to apply